autosklearn/metrics/__init__.py

```diff
@@ -104,19 +109,21 @@ def __call__(
         else:
             raise ValueError(type_true)
 
+        sc_args = {}  # type: Dict[str, Union[List[float], np.ndarray]]
```

Comment:
This should be named `kwargs`

Reply:
Also, what does the `sc` stand for?

Reply:
Sure, changed it to scorer_kwargs which should be more descriptive